### PR TITLE
TOOLS/gen-x11-icon.sh: use $(..) instead of legacy `..`

### PR DESCRIPTION
**Pull request by softfixes**

### CODE REVIEW

File: TOOLS/gen-x11-icon.sh [hunk 1]
```diff
@@ -4,8 +4,8 @@
 # etc/mpv-icon...), and it will write video/out/x11_icon.bin.
 
 conv() {
-    echo
-    echo `identify -format "icon: %w %h" "$1"`
+    printf "\n"
```
Comment (wiiaboo) [inline TOOLS/gen-x11-icon.sh]: Why not just pre-append the `\n` to the next line if you're converting the echos to printfs?

Comment (softfixes) [inline TOOLS/gen-x11-icon.sh]: Do you want me to do that?
I thought this way it looked a bit cleaner, but I can change.

File: TOOLS/gen-x11-icon.sh [hunk 2]
```diff
@@ -4,8 +4,8 @@
...
+    printf "$(identify -format "icon: %w %h" "$1")\n"
```
Comment (otommod) [inline TOOLS/gen-x11-icon.sh]: Shouldn't this better be `printf '%s\n' "$(identify ...)"` ?

**Comment by softfixes** [comment]
Applied both fixes, thanks for the tips.

**Comment by wiiaboo** [comment]
File was removed with https://github.com/mpv-player/mpv/commit/91279390710954519be79cff8388873888bf2f42.

**Pull request closed without merging**